eventstore/dynamodb/eventstore.go

```diff
@@ -1,373 +0,0 @@
-// Copyright (c) 2016 - The Event Horizon authors.
```

Comment:
Please consider keeping this copy header in the derived version, changing "The Event Horizon authors" to "The Event Horizon DynamoDB authors" and keeping an AUTHORS file in the root (with all current authors of the DynamoDB related changes in it).

Reply:
fixed here https://github.com/seedboxtech/eh-dynamo/commit/4ad7e11a39cbd22d0463d2fff271b3ee5a64cd9c